packages/grafana-data/src/transformations/transformers/reduce.test.ts

```diff
@@ -417,16 +417,16 @@ describe('Reducer Transformer', () => {
               "CA",
               "NY",
               "CA",
-              undefined,
+              ,
```

Comment:
What's the reasoning behind this change? I could see it being easy to miss an extra comma, and the `undefined` seems a bit easier to see

Reply:
it's a snapshot change, i guess jest 27 has changed how they store undefined in their snapshots 🤷